zio-http/shared/src/main/scala/zio/http/Body.scala

```diff
@@ -339,13 +346,36 @@ object Body {
   def fromStream[A](stream: ZStream[Any, Throwable, A])(implicit codec: BinaryCodec[A], trace: Trace): Body =
     StreamBody(stream >>> codec.streamEncoder, knownContentLength = None)
 
+  /**
+   * Constructs a [[zio.http.Body]] from stream of values based on a zio-schema
+   * [[zio.schema.codec.BinaryCodec]] and depends on `R`<br>
+   *
+   * Example for json:
+   * {{{
+   * import zio.schema.codec.JsonCodec._
+   * case class Person(name: String, age: Int)
+   * implicit val schema: Schema[Person] = DeriveSchema.gen[Person]
+   * val persons = ZStream(Person("John", 42))
+   * val body = Body.fromStream(persons)
+   * }}}
+   */
+  def fromStreamR[A, R](stream: ZStream[R, Throwable, A])(implicit codec: BinaryCodec[A], trace: Trace): RIO[R, Body] =
```

Comment:
This should be `fromStreamEnv` or `fromStreamEnvironment`, as we don't use the name of the type parameters in method names anymore (since ZIO 2.x).